receiver/filelogreceiver/go.mod

```diff
@@ -4,6 +4,7 @@ go 1.15
 
 require (
 	github.com/observiq/nanojack v0.0.0-20201106172433-343928847ebc
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.24.1-0.20210416221238-c731ffe72642 // indirect
```

Comment:
Needed to tweak this due to:

```
receiver/filelogreceiver$ go mod download ; go mod tidy
go: finding module for package github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/filestorage
go: downloading github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage v0.24.0
go: downloading github.com/open-telemetry/opentelemetry-collector-contrib v0.24.0
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/filelogreceiver imports
        github.com/open-telemetry/opentelemetry-collector-contrib/internal/stanza tested by
        github.com/open-telemetry/opentelemetry-collector-contrib/internal/stanza.test imports
        github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/filestorage: module github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage@latest found (v0.24.0), but does not contain package github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/filestorage
```

Reply:
No longer changed correct?

Reply:
Correct, this was resolved on main hence no longer needed here